debian/homestead.postinst

```diff
@@ -89,13 +89,6 @@ case "$1" in
 
         service clearwater-infrastructure restart
 
-        # Stop the cluster manager, so that it is restarted by Monit and picks
-        # up the new Cassandra scaling plugin. We check whether the process is 
-        # installed to avoid warning logs when etcd isn't being used
-        if [ -x "/etc/init.d/clearwater-cluster-manager" ]; then
-          service clearwater-cluster-manager stop || /bin/true
-        fi  
-
```

Comment:
Did you discuss this with Rob or somebody else? 
I don't know enough to say this is still needed, but given that these postinst files are going to be in place on -node nodes as well as nodes using the smaller packages, we may well want to keep this in.